File: python/PrimeNumber.py
import math

def prime(n):
 
    if (n <= 1):
        return False
 
    for i in range(2, int(math.sqrt(n))):
        if (n % i == 0):
            return False
 
    return True
 
 
if prime(21):
    print("Prime Number")
else:
    print("Non Prime")

File: c++/PrimeNumber.cpp
#include<iostream>
#include<cmath>
using namespace std;

int main(){
    int n = 0;
    cin>>n;
    bool flag = true;
    for(int i=2;i<=sqrt(n);i++){
        if(n%i == 0){
            cout<<"Non Prime";
            flag = false;
            break;
        }
    }

    if(flag){
        cout<<"Prime Number";
    }
    return 0;
}
   